examples/README.md

```diff
@@ -0,0 +1,36 @@
+This folder contains an example configuration file, as well as necessary files to run a demo of Transition. This demo corresponds to the territory of the Réseau de transport de Longueuil area.
+
+# Running the demo
+
+1. Make sure all the pre-requisites are installed. For complete installation instructions of all dependencies, see the [instructions for Ubuntu](../../docs/transition/installationUbuntu20.04.md).
```

Comment:
Not for now, but we'll need to update this. This is a bit of an outdated ubuntu version. We might need more generic instructions